packages/openapi-parser/src/utils/betterAjvErrors/helpers.ts

```diff
@@ -8,15 +8,15 @@ import {
   isEnumError,
   isRequiredError,
   notUndefined,
-} from './utils'
+} from './utils.ts'
 import {
   AdditionalPropValidationError,
   DefaultValidationError,
   EnumValidationError,
   PatternValidationError,
   RequiredValidationError,
   UnevaluatedPropValidationError,
-} from './validation-errors/index'
+} from './validation-errors/index.js'
```

Comment:
missed a js?

Reply:
```suggestion
} from './validation-errors/index.ts'
```

Reply:
Nah, it’s actually js, because this is just code I copied into the repository and never really migrated.